README.md

```diff
@@ -0,0 +1,92 @@
+<a name="readme-top"></a>
+
+  <h3><b>OOP-student-Library</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 OOP-student-Library ](#-oop-student-library-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 OOP-student-Library <a name="about-project"></a>
+
+**OOP-student-Library** .
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+VS Code, GIT, Ruby and NPM installed
+
+### Setup
+
+Clone this repository to your desired folder using the git clone command
+https://github.com/Roland-Ntwali/School-library-app.git
+
+### Usage
+
+Run the code in the console/irb.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Roland Ntwali**
+
+- GitHub: [Roland-Ntwali](https://github.com/Roland-Ntwali)
+- LinkedIn: [roland-ntwali](https://www.linkedin.com/in/roland-ntwali/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+To student success and coding partner
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
All the main headers from[ this README template](https://github.com/microverseinc/readme-template) should be present in a student's repo. The only optional section is FAQ.
- Kindly add the section Future Features, including 1-3 potential features to be added.
- Kindly add the section Key features describing the key features of your project.

- [ ] The license link you have attached is broken. Kindly add a valid link.